studio/src/studio.js

```diff
@@ -335,7 +332,7 @@ class MasStudio extends LitElement {
                 ? html`<mas-filter-panel></mas-filter-panel>`
                 : nothing}
             ${this.content} ${this.fragmentEditor} ${this.selectFragmentDialog}
-            ${this.toast} ${this.loadingIndicator} ${getOffferSelectorTool()}
+            ${this.toast} ${this.loadingIndicator}}
```

Comment:
There's an extra closing curly bracket at eol that should be removed.

Reply:
Good catch, thanks @Axelcureno